### ISSUE TITLE
New Rule: alpha-value-notation

### ISSUE THREAD

**Issue by azat-io** [eslint/css#75]
### Rule details

Specify percentage or number notation for alpha-values.

### What type of rule is this?

Warns about a potential problem

### Example code

There are two ways to set alpha notation.

Number:

```css
.test {
  color: rgb(0 0 0 / 0.5);
  opacity: 1;
}
```

Percentage:

```css
.test {
  color: rgb(0 0 0 / 50%);
  opacity: 100%;
}
```

I think it would be good to have a rule to check this.

```js
{
  rules: {
    'alpha-value-notation': ['error', 'percentage'],
  },
}
```

### Participation

- [x] I am willing to submit a pull request to implement this rule.

**Comment by azat-io** [comment]
No difference, just code style.

**Comment by nzakas** [comment]
As noted on https://github.com/eslint/css/issues/76, we aren't accepting stylistic rules into the plugin. You can always create your own plugin with these rules if you so desire.
